tests/src/system/basic/WskBasicTests.scala

```diff
@@ -348,14 +348,14 @@ class WskBasicTests
             assetHelper.withCleaner(wsk.rule, ruleName) {
                 (rule, name) =>
                     rule.create(name, trigger = triggerName, action = actionName)
-                    rule.create(name, trigger = triggerName, action = actionName, update = true)
+                    rule.create(name, trigger = triggerName, action = actionName, update = true, expectedExitCode = CONFLICT)
```

Comment:
As an additional tests, sure (since there's no way to create a test that's disabled).